### PR TITLE
Design changes

### PR DESCRIPTION
**Pull request by jillianchang**
I'm pretty happy with this! I didn't make many changes content wise- I basically took the existing components and made the interface look nicer. I really like the dynamic tabbed nav bar, which eliminates the need for URL redirects when navigating the tabs. I also spent many hours looking into how to preserve this Single Page Application nature for displaying the scanned lines, i.e. rewriting the current web page with the result.html contents without the page having to reload. Unfortunately, I wasn't able to find a solution to this, so my temporary solution was to display the scanned lines within the box section (which I did via template inheritance). It would be ideal if the form submission could occur without page reload, so it would be great if you could help tackle that (if the feat is accomplishable in the first place). 

I think my next goal is to improve the formatting of the scanned lines. I'm still thinking about what the most readable, clean  way would be to indicate the syllables and feet, along with the original/modified pronunciations. For instance, do we want to give users the option to select which pronunciations they want to display, or do we just want to choose one for standardization purposes? I thought of a few ideas for the structure. One is to display the original text on the left half, and the variable pronunciation on the right half with macrons/breves to indicate weight (and either D, S, T for the feet or color coding). Another idea is to display the original text and the variable pronunciation on the left half, and pure syllable structures on the right (e.g. ¯ ˘ ˘ ˈ ¯ ˘ ˘  ˈ ¯ ¯ ˈ ¯ ¯ ˈ ¯ ˘ ˘ ˈ ¯ ˚). 

Let me know what your thoughts are.

### CODE REVIEW

**Comment by jillianchang** [comment]
https://user-images.githubusercontent.com/57687959/143527270-013bbe43-03b9-4c85-b62d-35dd2beec84c.mov

Video demo of what it looks like

File: static/script.js
Comment (kylebgorman) [inline static/script.js]: do you want to get rid of this earlier version starting at line 15?

File: static/style.css
Comment (kylebgorman) [inline static/style.css:1]: Really glad you're involved in this, you know a lot more CSS than me, clearly ;)

Let me suggest:

1. find some CSS reflowing tool, and then run it on this file.
2. find some CSS minifying tool, then run it on this file to create `style.min.css`. 
3. check in both copies, but change links to be to the minified version from #2.

File: templates/base.html
Comment (kylebgorman) [inline templates/base.html]: no space after `=` here.

Possibly dumb question but can we load JQuery at the bottom, the way you loaded our other scripts? Or does it have to be loaded in `<head>`?

File: templates/base.html
Comment (kylebgorman) [inline templates/base.html:1]: There's some pretty weird indentation here. Maybe there's a reflowing tool that can rationalize this for us?

File: templates/index.html
Comment (kylebgorman) [inline templates/index.html]: final newline?

File: templates/result.html
Comment (kylebgorman) [inline templates/result.html]: final newline?

File: templates/result.html
Comment (kylebgorman) [inline templates/result.html]: Maybe "Back" or "Home"? IDK, "return to home" sounds weird to me but I can't say why?

File: app.py
Comment (kylebgorman) [inline app.py]: This isn't needed, is it? This file wasn't otherwise affected.

File: static/script.js
Comment (kylebgorman) [inline static/script.js:1]: Consider reflowing and minification as mentioned below.

**Review by kylebgorman — COMMENTED** [review]
Bravo! This looks very sharp. I have some nitpicks, but that's it.

Once you address those and this is merged, I can deploy to Heroku. (I tried to make it so that it would automatically deploy changes GitHub master branch but I'm not sure it worked; guess we'll find out.)

File: templates/result.html
Comment (jillianchang) [reply templates/result.html]: I changed it to "New Scan"- is that better?

File: static/style.css
Comment (jillianchang) [reply static/style.css:1]: I couldn't find a CSS reflowing tool, but I did minify both CSS and JS. What does the reflowing do?

Comment (kylebgorman) [reply static/style.css:1]: Just standardizes the spacing, indentation etc.  No worries then, I'll maybe do it a later date.

File: templates/base.html
Comment (kylebgorman) [inline templates/base.html]: This should be `style.min.css` now, as it should be in all the template files (and similarly in the static ones), otherwise we get no benefits from this.

**Review by kylebgorman — COMMENTED** [review]
Could you add some information to the README about how you minify? Or rather, how one would minify: what commands to run etc.

File: templates/base.html
Comment (jillianchang) [reply templates/base.html]: My bad, it was changed locally but I forgot to check it in.

**Comment by jillianchang** [comment]
> Could you add some information to the README about how you minify? Or rather, how one would minify: what commands to run etc.

I just used a website online that minifies (?)

**Comment by kylebgorman** [comment]
> I just used a website online that minifies (?)

ha nevermind then ;)

File: templates/result.html
Comment (kylebgorman) [reply templates/result.html]: Fine with me.

**Review by kylebgorman — COMMENTED** [review]
Looks to me like you just need to make sure everyone is using the minified CSS and we're good to go.

**Comment by jillianchang** [comment]
What’s your suggestion on what to say?

**Comment by kylebgorman** [comment]
> What’s your suggestion on what to say?

Re: minification? Nothing I guess, we can come back to that later.

**Comment by jillianchang** [comment]
> > What’s your suggestion on what to say?
> 
> Re: minification? Nothing I guess, we can come back to that later.

Okay, should we merge?

**Comment by kylebgorman** [comment]
The one issue I see before merging is that the static pages need to be
pointed at the style.min.css too, right?

**Comment by jillianchang** [comment]
About.html and input.html? Oh actually, I can remove those files because they're not in use. JS allows index.html to change div content upon clicking the navigation tabs.

**Comment by kylebgorman** [comment]
Yea, remove those and we should be good to go.

**Review by kylebgorman — APPROVED** [review]
LGTM, merge away

**Comment by jillianchang** [comment]
It says I don't have write access?

**Comment by kylebgorman** [comment]
Ah, I'll merge and then grant that to you.

**Comment by kylebgorman** [comment]
I just granted you write access but please use the fork-and-pull approach
all the same ;)

**Comment by jillianchang** [comment]
Yes, of course. It looks like it didn't automatically deploy to Heroku, guess it'll have to manually be done.

**Comment by jillianchang** [comment]
Never mind, I see it now :)

**Comment by kylebgorman** [comment]
Yeah, it takes a few minutes because it basically restarts the app from
scratch. I was actually logged into Heroku and it was sending me the logs
more or less live so I knew just went to press "refresh". Looks great in
person.

**Pull request merged**
